src/components/ebay-section-title/examples/05-info/template.marko

```diff
@@ -0,0 +1,10 @@
+<ebay-section-title>
+    <ebay-section-title-title>Today’s Deals – All With Free Shipping</ebay-section-title-title>
+    <ebay-section-title-subtitle>Plus, guaranteed best prices.</ebay-section-title-subtitle>
+    <ebay-section-title-info>
```

Comment:
Does this have different markup? Can we just put it inside the title or something?

Reply:
I tried this, but since I use a `<div class="section-title__info>` container around infotip, the container class still exists when infotip isn't passed in body (data.renderBody is always defined, even when infotip isn't passed).

Possible workaround: I can get rid of "section-title__info" container, and possibly "section-title__overflow" container as well. Instead, I would style the infotip and overflow directly like `.section-title .infotip` and `.section-title .menu-button`. Then `<ebay-section-title-info>` and `<ebay-section-title-overflow>` won't be required anymore. If I do this, I need to change Skin as well, so checking with team first.

Thoughts?

Reply:
@dkallus @agliga My thinking was that the containers would give us a bit more flexibility in terms of layout. If you think they are redundant and provide no benefit, then perhaps we could first merge your current skin PR, then you could post a follow up draft showing the proposed change without container elements?

Reply:
I'd prefer to keep the containers